packages/artillery-plugin-apdex/index.js

```diff
@@ -15,6 +15,53 @@ class ApdexPlugin {
   constructor(script, _events) {
     this.script = script;
 
+    if (
```

Comment:
Better to return early here if conditions are met. That makes the diff much smaller too as most of the existing code stays in place rather than getting moved around.

Reply:
Unless I'm missing something, I can't do an early return, as something always needs to run inside the if. Half the logic needs to run in workers, the other half in the main thread.

But I reversed the if condition anyway, as I do think checking for workers instead makes it easier to read, is more in-line with other plugins and makes for a smaller diff like you said 👍 .